apps/files/src/mixins.js

```diff
@@ -447,9 +447,23 @@ export default {
         // FIXME: this might break if relativePath is not the currentFolder
         // and is a mount point that has no chunk support
         if (this.browserSupportsChunked && this.currentFolder.isChunkedUploadSupported) {
+          let chunkSize = this.configuration.uploadChunkSize
+          console.log('Configured chunk size:', chunkSize)
+          if (this.capabilities.files.tus_support.max_chunk_size > 0) {
+            if (
+              chunkSize === null ||
+              chunkSize === 0 ||
+              chunkSize > this.capabilities.files.tus_support.max_chunk_size
+            ) {
+              chunkSize = this.capabilities.files.tus_support.max_chunk_size
```

Comment:
Could this be shortened with `Math.max()`?

Reply:
Ah, it would return the wrong one in case of 0 or null. Nevermind 🤦

Reply:
please recheck, I managed to find a way: converting null to 0.

0 means "unlimited" so the logic can still be summarized with a Math.max

Reply:
I mixed up max and min... we do want the admin to override and specify a smaller chunk size than the advertised one

Reply:
I've reverted now to the logic we had in the first iteration

Reply:
Yeah, I mixed them as well when I wrote the first comment (realised after it and edited it)